gpflow/kernels/linears.py

```diff
@@ -13,24 +13,30 @@ class Linear(Kernel):
     where σ²  is the variance parameter.
     """
 
-    def __init__(self, variance=1.0, active_dims=None, ard=None):
+    def __init__(self, variance=1.0, active_dims=None):
         """
         - input_dim is the dimension of the input to the kernel
         - variance is the (initial) value for the variance parameter(s)
-          if ard=True, there is one variance per input
+          defaults to 1.0 but supports ARD i.e. one variance per input.
```

Comment:
Could we have a more complete explanation? e.g. if you want ARD but initialised to 1 you'd have to pass in [1, 1, ..., 1] (where the length of the array is equal to the input dimension)

Reply:
(Applies to all kernels with ard feature)

Reply:
Makes sense - have done this